tests/pthread/call_sync_on_main_thread.c

```diff
@@ -19,15 +20,15 @@ extern int receivesAndReturnsAnInteger(int i);
 
 int main()
 {
-	char dst[256];
-	char name[7] = "resize";
-	getDomElementParentInnerHTML(name, dst, sizeof(dst));
+	char name[7] = "body";
+	const char *dst = getDomElementContents(name);
 	memset(name, 0, sizeof(name)); // Try to uncover if there might be a race condition and above line was not synchronously processed, and we could take name string away.
```

Comment:
are the changes to this file necessary? (comment above said they were, but I don't see how?)

Reply:
The old test asserted the existence of an inner HTML text node with text contents "resize" in it (that was the [ ] resize canvas checkbox in default shell). In MINIMAL_RUNTIME shell such checkbox does not exist, so I replaced it with a string snippet `<canvas` that can be found in both shells.